.github/workflows/build.yml

```diff
@@ -0,0 +1,125 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  dotnetBuildConfiguration: 'Release'
+  dotnetVerbosity: 'Detailed'
+
+jobs:
+  package_nuget:
+    name: Build and pack NuGet packages
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            include:
+            - os: windows-latest
+              artifactName: activelogin-identity-nuget-windows
+            - os: macos-latest
+              artifactName: activelogin-identity-nuget-macos
+            - os: ubuntu-latest
+              artifactName: activelogin-identity-nuget-ubuntu
+    env:
+       sourceRepositoryBranch: $env:GITHUB_REF_NAME
+       sourceRepositoryCommit: $env:GITHUB_SHA
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.7.2
+      
+      - name: Dotnet restore
+        run: dotnet restore --verbosity ${{ env.dotnetVerbosity }}
+        shell: pwsh
+      
+      - name: Dotnet build
+        run: dotnet build --configuration ${{ env.dotnetBuildConfiguration }} --verbosity ${{ env.dotnetVerbosity }}
+        shell: pwsh
+
+      - name: 'Dotnet pack: ci'
+        run: |
+          dotnet pack ./src/ActiveLogin.Identity.Swedish/ActiveLogin.Identity.Swedish.fsproj --output ${{ env.ciOutputFolder }} --configuration ${{ env.dotnetBuildConfiguration }} --verbosity ${{ env.dotnetVerbosity }} /p:Configuration=${{ env.dotnetBuildConfiguration }} /p:RepositoryBranch=${{ env.sourceRepositoryBranch }} /p:RepositoryCommit=${{ env.sourceRepositoryCommit }} /p:VersionSuffix=ci-$env:GITHUB_RUN_ID
```

Comment:
What’s the difference between `—configuration` and the msbuild property `/p:Configuration`? Are they both required?